### PR TITLE
Directory error handling

### PR DESCRIPTION
**Pull request by haeungun**
Now, An error will occur if the directory is not selected.
So handled errors that occurred when the user did not select a directory.

ISSUE=none

### CODE REVIEW

**Comment by byeolbit** [comment]
lgtm

File: src/app/repository/repository.component.ts [hunk 1]
```diff
@@ -43,47 +43,51 @@ export class RepositoryComponent implements OnInit{
...
-    openProject.then(this.cloneRepo)
-               .then(this.runJekyll)
-               .then(result => this.navigateToEditor(result));
-  }
-
-  selectRepo(event) {
-    let target = event.target.innerText.trim();
-    if (target) {
-      this.selectedRepo = target;
-    }
```
Comment (byeolbit) [inline src/app/repository/repository.component.ts]: selectRepo function is need to select repository from repository list.

File: src/app/repository/repository.component.ts [hunk 2]
```diff
@@ -43,47 +43,51 @@ export class RepositoryComponent implements OnInit{
...
-        user: user
-      })
+                          repo:string, user:string): Promise<any> {
+    return new Promise((resolve, reject) => {
+      const dialog = remote.dialog;
+      const directory = dialog.showOpenDialog({properties: ['openDirectory']});
+      
+      if (directory) {
+        resolve({
+          path: dialog.showOpenDialog({properties: ['openDirectory']}).toString(),
```
Comment (byeolbit) [inline src/app/repository/repository.component.ts]: I think this code calls dialog again.
How about change this to `path: directory.toString()`?

Comment (haeungun) [reply src/app/repository/repository.component.ts]: Oh, you're right.
I missed it.

**Comment by byeolbit** [comment]
lgtm :)

**Pull request merged**